tests/database/test_unittestdb.py

```diff
@@ -0,0 +1,96 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of `ensembl.utils.database.unittestdb` module."""
+
+from contextlib import nullcontext as does_not_raise
+from pathlib import Path
+from typing import ContextManager, Optional
+
+import pytest
+from pytest import FixtureRequest, param, raises
+from sqlalchemy_utils.functions import database_exists
+
+from ensembl.utils.database import UnitTestDB
+
+
+class TestUnitTestDB:
+    """Tests `UnitTestDB` class.
+
+    Attributes:
+        dbs: Dictionary of `UnitTestDB` objects with the database name as key.
+
+    """
+
+    dbs: dict[str, UnitTestDB] = {}
+
+    @pytest.mark.dependency(name="test_init", scope="class")
+    @pytest.mark.parametrize(
+        "src, name, expectation",
+        [
+            param(Path("mock_db"), None, does_not_raise(), id="Default test database creation"),
+            param(Path("mock_db"), "renamed_db", does_not_raise(), id="Rename test database"),
```

Comment:
```suggestion
            param(Path("mock_db"), "renamed_db", does_not_raise(), id="Rename test database"),
            param(Path("mock_db"), "", does_not_raise(), id="Empty name, use default"),
```

Reply:
This is equivalent to using `None`, maybe it may be worth changing the id of the first parametrization instead?